cmd/helm/create_test.go

```diff
@@ -143,8 +143,9 @@ func TestCreateStarterCmd(t *testing.T) {
 		t.Errorf("Wrong API version: %q", c.Metadata.ApiVersion)
 	}
 
-	if l := len(c.Templates); l != 7 {
-		t.Errorf("Expected 6 templates, got %d", l)
+	expectedtemplatecount := 6
```

Comment:
I feel like this should be 7 and line 147 should just be comparing against the value of `expectedtemplatecount`. The original test output was incorrect.

Also, the general Go convention is to use camelCase for variable naming, so if you could change this to `expectedTemplateCount := 7`, that'd be great. Thanks

Reply:
I agree - I had worried about that check! Will fix both things you suggest and push back up.